pages/common/fish.md

```diff
@@ -0,0 +1,27 @@
+# fish
+
+> A command-line shell designed to be user firendly.
```

Comment:
Maybe mention the acronym in the first line. And then give the details. How about 
```
> Friendly Interactive SHell.
> A user friendly command line interpreter.
```

Reply:
Agreed, changed.